frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnField.tsx

```diff
@@ -32,7 +35,15 @@ const MultiColumnField: React.FC<MultiColumnFieldProps> = ({
           helperText={helpText}
           isRequired={required}
         >
-          <MultiColumnFieldHeader headers={headers} />
+          {fieldValue.length < 1 ? (
+            emptyMessage && (
+              <div className="odc-multi-column-field__empty-message">
+                <SecondaryStatus status={emptyMessage} />
+              </div>
+            )
+          ) : (
+            <MultiColumnFieldHeader headers={headers} />
+          )}
```

Comment:
@openshift/team-devconsole-ux 

As it stands, without an "empty message" will remove the headers. I assume this desirable? 

![Screen Shot 2020-02-11 at 8 10 36 AM](https://user-images.githubusercontent.com/8126518/74239551-10e5d000-4ca6-11ea-9970-b8f35a0ae9d7.png)

![Screen Shot 2020-02-11 at 8 10 23 AM](https://user-images.githubusercontent.com/8126518/74239548-104d3980-4ca6-11ea-89cf-63f2b4f07214.png)

--- 

We use this component in 4 spots

- Pipeline Parameters
- Pipeline Resources
- Project Access
- Knative Traffic Split

We'll need to come up with how we want to handle those. I logged https://issues.redhat.com/browse/ODC-2976 a while back to address chasing them down and asking this question.